src/views/wallet/Crypto/index.js

```diff
@@ -14,21 +14,34 @@ import DataGridTransaction from '../../../components/wallet/crypto/Transactions/
 
 function WalletCrypto() {
     const [transaction, setTransaction] = useState({ isEnabled: false, coin: {} });
+    const [coins, setCoins] = useState(coinsData);
 
     const transactionHandle = (event, data) => {
         setTransaction({ isEnabled: true, coin: data.row });
     };
     const backClickHandle = () => {
         setTransaction({ ...transaction, isEnabled: false });
     };
-
+    const handleOnDelete = (e, clickedCoin) => {
+        const newCoins = coins.filter((coin) => coin.id !== clickedCoin.id);
+        setCoins(newCoins);
+    };
+    const addTransactionHandle = (e, clickedCoin) => {
+        console.log(clickedCoin);
+        console.log(coins);
+        const index = coins.findIndex((coin) => coin.id === clickedCoin.id);
+        console.log(index);
+        if (index !== 0) {
```

Comment:
to avoid duplicates, consider wrapping this with an IF, checking if the coin you want to add is not already added